app/src/components/Stat/Stat.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { StatProps } from './types';
+
+export const Stat = ({
+  title,
+  value,
+  style = 'stats shadow',
+  align = 'stat text-center',
+  titleStyle = 'stat-title',
+  valueStyle = 'stat-value',
```

Comment:
If you pass in props for any of these guys  the default values will be overwritten. Try to use the other components as a guide. Button is probably a good one. We should have base classes that can be extended.

Reply:
It was actually my intention to leave those by default and use the Stat display component as it is and if needed you could make some changes by altering the default values. I did look at the Button and thought I was following it as a guide as in:
`export const Button = ({
  title,
  size = 'md',
  color = 'primary',
  variant = 'solid',
`
talk soon :)